src/ParameterStoreBuildWrapperPlugin.groovy

```diff
@@ -48,6 +59,7 @@ class ParameterStoreBuildWrapperPlugin implements TerraformEnvironmentStagePlugi
         def parameterStoreOptions = defaultOptions + options
 
         return { closure ->
+            // sh "echo ${parameterStoreOptions}" //DEBUG
```

Comment:
Let's remove this, and make an issue to generally support informational and debugging messages.  This isn't the only change that would benefit from the  ability to toggle "logging" on and off.

Reply:
kk captured it here #304